### ISSUE TITLE
Sentence on "ARIA: suggestion role" page is incomplete

### ISSUE THREAD

**Issue by OwenEdwards** [mdn/content#34567]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/suggestion_role

### What specific section or headline is this issue about?

Examples

### What information was incorrect, unhelpful, or incomplete?

The following sentence is incomplete:

> Browsers tend to provide a default black strikethrough for deletions, and a black underline for insertions, but you'll probably want to use

https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/suggestion_role/index.md?plain=1#L42

### What did you expect to see?

Browsers tend to provide a default black strikethrough for deletions, and a black underline for insertions, but you'll probably want to use **CSS to customize the visual styling of deletions and insertions.**
